pepys_import/core/store/data_store.py

```diff
@@ -590,8 +589,43 @@ def search_table_type(self, table_type_id):
 
     #############################################################
     # New methods
+    def find_datafile(self, datafile_name):
+        """
+        This method tries to find a Datafile entity with the given datafile_name. If it
+        finds, it returns the entity. If it is not found, it searches synonyms.
 
-    def get_datafile(self, datafile_name, datafile_type):
+        :param datafile_name:  Name of Datafile
+        :type datafile_name: String
+        :return:
+        """
+        datafile = (
+            self.session.query(self.db_classes.Datafile)
+            .filter(self.db_classes.Datafile.reference == datafile_name)
+            .first()
+        )
+        if datafile:
+            return datafile
+
+        synonym = (
+            self.session.query(self.db_classes.Synonym)
+            .filter(
+                self.db_classes.Synonym.synonym == datafile_name,
+                self.db_classes.Synonym.table == "Datafiles",
```

Comment:
@BarisSari  - I don't know if this doesn't matter in Python, but this is an example of a _Magic Number_. (well, String)  Normal programming practice is to avoid this, and to only declare it as a constant, in one place.  Actually, we may already have it declared somewhere as a constant.  When it's a constant, we know we only have to change it in one place, for it to be updated everywhere.

Reply:
There is no `const` or any similar keywords in Python. However, you're right that we should avoid this. Therefore, I did the following changes:
- Created a new file under **_pepys_import/core/store_**, named **_constants.py_**
- Used **final** hint to tell developers that it shouldn't be reassigned even though it doesn't prevent change.
- Changed all strings passed to **add_to_synonyms** method with the values in **_constants.py_**. 
- Changed all table names in DB Classes with the values in **_constants.py_**.

https://github.com/debrief/pepys-import/pull/73/commits/b98898d73f947a7c5af5fefa0a21c7f3a44bc4fd